Gemfile

```diff
@@ -1,15 +1,18 @@
 source 'https://rubygems.org'
 
-# Gems here are used to run `rake release`.
+group :release do
+  gem 'emeril'
+end
 
-gem 'emeril'
 gem 'rake'
 gem 'chef', '= 14.10.9'
-gem 'foodcritic', '= 11.4.0'
+gem 'foodcritic', '~> 11.4.0'
+gem 'cucumber-core', '~> 3.2.1'
```

Comment:
As foodcritic is laxist on its dependencies, if we don't fix the cucumber version, it breaks the foodcritic requirements.